server.js

```diff
@@ -620,10 +620,30 @@ function getConfigData(req) {
     }/auth/realms/openshift/protocol/openid-connect/logout?redirect_uri=${logoutRedirectUri}',
     threescaleWildcardDomain: '${process.env.THREESCALE_WILDCARD_DOMAIN || ''}',
     integreatlyVersion: '${process.env.INTEGREATLY_VERSION || ''}',
-    clusterType: '${process.env.CLUSTER_TYPE || ''}'
+    clusterType: '${process.env.CLUSTER_TYPE || ''}',
+    optionalWatchServices: ${arrayFromString(process.env.OPTIONAL_WATCH_SERVICES, ',')},
+    optionalProvisionServices: ${arrayFromString(process.env.OPTIONAL_PROVISION_SERVICES, ',')}
   };`;
 }
 
+function arrayFromString(data, sep) {
```

Comment:
I had to write this function since the value needs to be a valid array in string formatting

Reply:
Why not something like:

```js
const arrayFromString = (data, sep) => data.split(sep).filter(item => item !== '');
```

Reply:
would that parse the array in string format? I got some problems using an array object within that formatted string

Reply:
yes, if the string is in a format like `a,b`. An empty string would return an empty array.

Reply:
I mean `['mdc']`  was being parsed as "mdc", let me give it  a try again